src/core/namespace.c

```diff
@@ -74,7 +74,17 @@ typedef struct TargetMount {
 static const TargetMount protect_kernel_tunables_table[] = {
         { "/proc/sys",                  READONLY,       false, },
         { "/proc/sysrq-trigger",        READONLY,       true, /* Not always compiled into kernel */ },
+        { "/proc/latency_stats",        READONLY,       true, },
+        { "/proc/mtrr",                 READONLY,       true, },
+        { "/proc/apm",                  READONLY,       true, },
+        { "/proc/acpi",                 READONLY,       true, },
+        { "/proc/timer_stats",          READONLY,       true, },
+        { "/proc/asound",               READONLY,       true, },
+        { "/proc/bus",                  READONLY,       true, },
+        { "/proc/fs",                   READONLY,       true, },
+        { "/proc/irq",                  READONLY,       true, },
         { "/sys",                       READONLY,       false, },
+        { "/sys/kernel/debug",          READONLY,       true, },
```

Comment:
hehe, yep! I added the entry now, but actually Lennart already added that item about tracing `ProtectTracing=` in TODO https://github.com/systemd/systemd/pull/4185/commits/842bb2f9c6da1b34e77f91e44fd4325d97b4566d , and I have that in a separate patch that's why... thanks